build.gradle

```diff
@@ -135,6 +152,11 @@ project(':iceberg-data') {
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     }
   }
+
+  test {
+    // Only for TestSplitScan as of Gradle 5.0+
+    maxHeapSize '1500m'
```

Comment:
I verified this on master - if you just change the Gradle version to 5.0+, the `TestSplitScan` test fails with the same error. So the memory settings must have changed in Gradle 5.0. Nothing that was changed with regards to style should have had an effect on the memory usage.